ang/taskRunner.js

```diff
@@ -66,43 +65,49 @@
         return;
       }
 
-      var data = {}
-      data['task_id'] = taskId;
+      const data = {
+        task_id: taskId,
+        async: true,
```

Comment:
This should only be true if the `enableBackgroundQueue` Civi setting is enabled.